content/en/docs/topics/registries.md

```diff
@@ -11,17 +11,16 @@ stored and shared across OCI-based registries.
 
 ## Enabling OCI Support
 
-Currently OCI support is considered *experimental*.
+Prior to Helm v3.8.0, OCI support was considered *experimental* and needed to be
+enabled. As of v3.8.0 it is enable by default.
 
-In order to use the commands described below, please set `HELM_EXPERIMENTAL_OCI`
-in the environment:
+To enable the OCI experiment for Helm versions prior to v3.8.0, please set
```

Comment:
> To enable the OCI experiment for Helm versions prior to v3.8.0

Maybe we want to say "To enable OCI experimental support for Helm versions prior to v3.8.0"? This is fine as it is, but my edit may make it a little clearer.

Reply:
done